stats/stats.go

```diff
@@ -34,28 +37,26 @@ func SetContainerName(name string) { containerName = name }
 type CounterType struct {
 	disk   string
 	client int64
-	name   string
+	op     string
 }
 
-func Counter(name string) CounterType { return CounterType{name: name} }
-
-func (c CounterType) Disk(disk string) CounterType {
-	c.disk = disk
-	return c
+func Counter(disk, op string) *CounterType {
+	return &CounterType{
+		op:   op,
+		disk: disk,
+	}
 }
 
-func (c CounterType) Client(id int64) CounterType {
+func (c *CounterType) Client(id int64) *CounterType {
 	c.client = id
 	return c
 }
 
-func (c CounterType) Add() {
-	var prefix string
-	if c.disk != "" {
-		prefix = c.disk + "_"
+func (c *CounterType) Add() {
+	metrics.Counter(c.disk + "_" + c.op).Add()
+	if c.client != 0 {
+		metrics.Counter(clientCounterPrefix + strconv.FormatInt(c.client, 16)).Add()
 	}
-	metrics.Counter(prefix + c.name).Add()
-	metrics.Counter(strconv.FormatInt(c.client, 16) + "_ops").Add()
```

Comment:
enforce pkg is relying on this format now: https://github.com/c-fs/cfs/blob/master/enforce/enforce.go#L51
Could we make them consistent?